data_managers/data_manager_rgi_build_db/data_manager/rgi_database_builder.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<tool id="rgi_database_builder" name="rgi_database_builder" tool_type="manage_data" version="1.0.0">
+    <description>database builder</description>
+    <requirements>
+        <requirement type="package" version="4.2.2">rgi</requirement>
+    </requirements>
+    <stdio>
+        <exit_code description="Error" level="fatal" range="1:" />
+    </stdio>
+    <command interpreter="python">
+    <![CDATA[
+        rgi_database_builder.py --out "${out_file}"
+        #if $name:
+            --name "${name}"
+        #end if
+        #if $url:
+            --url "${url}"
+        #end if        
+    ]]>
+    </command>
+    <inputs>
+        <param name="name" type="text" value="" label="Name for this database" help="Name for this database, or leave blank for today's date" optional="True"/>
+        <param name="url" type="text" value="" label="Url" help="Url for the CARD data e.g https://card.mcmaster.ca/download/0/broadstreet-v3.0.0.tar.gz" optional="True"/>
```

Comment:
I believe if you set something like `value="https://card.mcmaster.ca/latest/data"` then this parameter will default to this value (which should be the latest database).

In the **help** section could you also link to https://card.mcmaster.ca/download which is where someone would go to search for previous CARD releases.